backends/dpdk/control-plane/bfruntime_arch_handler.h

```diff
@@ -195,8 +197,8 @@ class BFRuntimeArchHandlerPSA final : public P4RuntimeArchHandlerCommon<Arch::PS
         selector.set_num_groups(actionSelector.numGroups);
         p4configv1::ActionProfile profile;
         profile.set_size(actionSelector.size);
-        auto tablesIt = actionProfilesRefs.find(actionSelector.name);
-        if (tablesIt != actionProfilesRefs.end()) {
+        auto tablesIt = this->actionProfilesRefs.find(actionSelector.name);
```

Comment:
is this required?

Reply:
Yes. this pointer is required here as actionProfilesRefs is not dependent on the template parameter  and non-dependent names in a template function should be accessed using 'this' or 'using'. 

More about dependent name here: https://en.cppreference.com/w/cpp/language/dependent_name
https://blog.llvm.org/2009/12/dreaded-two-phase-name-lookup.html